src/assets/i18n/en.json5

```diff
@@ -3893,6 +3895,7 @@
 
   "submission.sections.upload.upload-successful": "Upload successful",
 
+  "submission.sections.accesses.form.discoverable-label": "Discoverable",
```

Comment:
We should add a `help` label for this `Discoverable` checkbox, otherwise it's very likely to be misunderstood.  I'd recommend adding help text under it that says something like:
"When checked, this item will be discoverable in search/browse. When unchecked, the item will only be available via a direct link and will not appear via search/browse."